xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodec.cpp

```diff
@@ -81,5 +81,31 @@ VideoPicture& VideoPicture::SetParams(const VideoPicture &pic)
   return *this;
 }
 
+bool VideoPicture::CompareDisplayMetadata(const VideoPicture& pic) const
+{
+  if (this->hasDisplayMetadata != pic.hasDisplayMetadata)
+    return false;
+
+  if (!pic.hasDisplayMetadata)
+    return true;
```

Comment:
Comment added: not are unknown but not has metadata (e.g. SDR video) and is not need compare

> A reason for this-> instead of just using the member?

For nothing special, seems more clearer to me.